README.md

```diff
@@ -210,16 +210,6 @@ The following is a list of additional third-party dependencies used as part of t
 
 The following is a list of additional third-party dependencies used as part of the CI/CD workflows. These are not present in the final packaged application.
 
-- **Danger**
```

Comment:
⛏️ Could include Codecov in the list of CI/CD dependencies.

Reply:
We actually don't note this anywhere else.